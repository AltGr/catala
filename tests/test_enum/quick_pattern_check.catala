@Article@

/*
new enum E:
  -- Case1 content int
  -- Case2
  -- Case3

new scope A:
  param x content E
  param y content bool
  param z content bool
  param t content bool 

scope A:
  def x := Case1 content 2
  def y := x with Case1
  def z := x with Case2
  def t := x with Case3
*/
