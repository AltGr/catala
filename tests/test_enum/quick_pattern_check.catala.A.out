[RESULT] Computation successful! Results:
[RESULT] t = false
[RESULT] x = Case1 2
[RESULT] y = true
[RESULT] z = false
